src/components/course/data/utils.jsx

```diff
@@ -154,6 +155,33 @@ export function getProgramIcon(type) {
 
 export const numberWithPrecision = (number, precision = 2) => number.toFixed(precision);
 
+/**
+ * Displays content price with precision as a range or singular price
+ * @param priceRange
+ * @returns {*|string}
+ */
+export const getContentPriceDisplay = (priceRange) => {
+  if (!priceRange?.length) {
+    return numberWithPrecision(ZERO_PRICE);
+  }
+  const minPrice = Math.min(...priceRange);
+  const maxPrice = Math.max(...priceRange);
+  if (maxPrice !== minPrice) {
+    return `${numberWithPrecision(minPrice)} - ${numberWithPrecision(maxPrice)}`;
+  }
+  return numberWithPrecision(priceRange[0]);
+};
+
+export const formatPrice = (price, options = {}) => {
```

Comment:
[nit] Maybe move to a more global utils file so it can be used outside of the `course` module.

Reply:
It looks like it's used anywhere yet; is this intentional?

Reply:
Updated and moved to the global utils file. Also began utilizing it. It was brought in with the intention of utilizing it.